lib/ruby_lsp/listeners/document_symbol.rb

```diff
@@ -83,6 +86,19 @@ def on_call_node_enter(node)
           handle_attr_accessor(node)
         elsif node.name == :alias_method
           handle_alias_method(node)
+        elsif node.name == :namespace
```

Comment:
Since we now have so many cases doing `node.name`, can we please extract it into a local variable before the `if`?

Reply:
I've update an update for that.